streams/src/main/java/org/apache/kafka/streams/processor/internals/TaskManager.java

```diff
@@ -243,18 +242,24 @@ public void handleAssignment(final Map<TaskId, Set<TopicPartition>> activeTasks,
 
         for (final Task task : tasksToClose) {
             try {
-                if (task.isActive()) {
```

Comment:
As we discussed in the other PR, I'm removing the logic for TM to check on task state and replaced it with checking that there should be nothing to commit (since suspend / pre-commit / post-commit are not all idempotent).

Reply:
What other PR?

Reply:
This is the PR from @ableegoldman https://github.com/apache/kafka/pull/8962